CHANGELOG.md

```diff
@@ -8,6 +8,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased] - ReleaseDate
 
+### Added
+- **BREAKING** init,configurer: allow to check configs without starting the system ([#95]).
```

Comment:
Why `BREAKING`?

Reply:
The only thing is breaking here: moving `start` and `try_start` to the `init` submodule. It should be noted separately under the `Changed` section with the `BREAKING` disclaimer.